charts/gardener-dashboard/values.yaml

```diff
@@ -315,7 +315,8 @@ global:
     #   authentication:
     #     username: dashboard
     #     token: dummytoken
-    #   # pollIntervalSeconds: 30 # only necessary when dashboard's webhook can't be reached by github and thus polling needs to be done
+    #   pollIntervalSeconds: 900 # In which interval GitHub issues will be synced. Timer is resetted by receiving a webhook.
+    #   syncThrottleSeconds: 20 # Max number of syncs in a given time window to avoid issues with rate limiting. Default 20.
```

Comment:
```suggestion
    #   syncThrottleSeconds: 20 # Defines the time window in which only one sync is allowed to avoid issues with rate limiting. Default 20.
```